### PR TITLE
Update (durability): Update the descriptions

### PR DESCRIPTION
**Pull request by Meggielqk**

### CODE REVIEW

File: en_US/durability/durability_introduction.md [hunk 1]
```diff
@@ -12,83 +12,81 @@ This feature is available starting from EMQX v5.7.0. However, it does not yet su
...
 
-### Types of Client Sessions
+### Sessions and Durable Storage
 
-The durable sessions in EMQX only applies to persistent sessions, so it's necessary to first understand the categorization of MQTT client sessions.
+**Session**: A session is a lightweight process within EMQX created for every client connection. Sessions implement behaviors prescribed to the broker by MQTT Standard, including initial connection, subscribing and unsubscribing to topics, and message dispatching.
 
-According to the MQTT standard, client sessions facilitate the management of client connections and states within the MQTT broker. Informally, EMQX separates client sessions into 2 logical categories:
+**Durable Storage**: Durable storage is an internal database within EMQX. Sessions may use it to save their state and MQTT messages sent to the topics. Database engine powering durable storage uses [RocksDB](https://rocksdb.org/) to save the data on disk, and [Raft](https://raft.github.io/) to consistently replicate data across the cluster. It is important not to confuse durable storage with **Durable Sessions**.
```
Comment (ieQu1) [inline en_US/durability/durability_introduction.md]:
```suggestion
**Durable Storage**: Durable storage is an internal database within EMQX. Sessions may use it to save their state and MQTT messages sent to the topics. Database engine powering durable storage uses [RocksDB](https://rocksdb.org/) to save the data on disk, and [Raft algorithm](https://raft.github.io/) to consistently replicate data across the cluster. It is important not to confuse durable storage with **Durable Sessions**.
```

File: en_US/durability/durability_introduction.md [hunk 2]
```diff
@@ -112,7 +110,7 @@ Even if durable sessions are not enabled, following steps 2-4 will still retain
 
    Restart EMQX to apply the configuration.
 
-2. Adjust MQTT client connection parameters to enable persistent sessions.
+2. Adjust MQTT client connection parameters to implement durable sessions.
```
Comment (ieQu1) [inline en_US/durability/durability_introduction.md]:
```suggestion
2. Adjust MQTT client connection parameters to activate session durability.
```

Comment (ieQu1) [reply en_US/durability/durability_introduction.md]: "Activate" or "enable".

File: zh_CN/durability/durability_introduction.md [hunk 3]
```diff
@@ -112,7 +107,7 @@ durable_sessions {
 
 重启 EMQX 以应用配置。
 
-### 2. 调整 MQTT 客户端连接参数以启用持久会话
+### 2. 调整 MQTT 客户端连接参数以采用持久会话
```
Comment (Meggielqk) [inline zh_CN/durability/durability_introduction.md]:
```suggestion
### 2. 调整 MQTT 客户端连接参数以启用持久会话
```

**Review by ieQu1 — APPROVED** [review]

**Review by wivwiv — APPROVED** [review]

**Pull request merged**